guides/doc-Upgrading_and_Updating/topics/upgrading_capsule_server.adoc

```diff
@@ -52,8 +62,21 @@ On the main {Project} server:
                        --certs-tar "~/myproxy.example.com-certs.tar"
 ----
 +
-For more information on custom SSL certificates signed by a Certificate Authority, see {InstallingSmartProxyDocURL}deploying-a-custom-ssl-certificate-to-{smart-proxy-context}-server_{smart-proxy-context}[Deploying a Custom SSL Certificate to {SmartProxyServer}] in _{InstallingSmartProxyDocTitle}_.
+
+. If you use custom SSL certificates on the {smart-proxy-context} server, use the following command to regenerate the certificate tar file on the {Project} server:
 +
+[options="nowrap" subs="attributes"]
+----
+# {smart-proxy-context}-certs-generate --foreman-proxy-fqdn "myproxy.example.com" \
+    --certs-tar "~/myproxy.example.com-certs.tar" \
+    --server-cert "/root/{smart-proxy-context}_cert/{smart-proxy-context}_cert.pem" \
+    --server-key "/root/{smart-proxy-context}_cert/{smart-proxy-context}_cert_key.pem" \
+    --server-ca-cert "/root/{smart-proxy-context}_cert/ca_cert_bundle.pem" \
+    --certs-update-server
```

Comment:
```suggestion
# {smart-proxy-context}-certs-generate \
--foreman-proxy-fqdn "myproxy.example.com" \
--certs-tar "~/myproxy.example.com-certs.tar" \
--server-cert "/root/{smart-proxy-context}_cert/{smart-proxy-context}_cert.pem" \
--server-key "/root/{smart-proxy-context}_cert/{smart-proxy-context}_cert_key.pem" \
--server-ca-cert "/root/{smart-proxy-context}_cert/ca_cert_bundle.pem" \
--certs-update-server
```

Please also use attributes instead of "myproxy".

Reply:
same as above, 
No idea which attribute should be used here. I checked  all the attribute files, but I did not get any attribute for myproxy.

Reply:
You can use "smartproxy-example-com":

```
guides/common/attributes-base.adoc:94::smartproxy-example-com: smartproxy.example.com
guides/common/attributes-orcharhino.adoc:44::smartproxy-example-com: orcharhino-proxy.example.com
guides/common/attributes-satellite.adoc:85::smartproxy-example-com: capsule.example.com
```

Reply:
There are four more instances of **myproxy.example.com**, shall we replace them with **smartproxy-example-com** too? 
I am asking this as the original file (procedure) not written by me.

Reply:
Yes, that would be much appreciated.